kafka_scripts/ksql_scripts.py

```diff
@@ -0,0 +1,149 @@
+import requests
+import json
+
+from .constants import KafkaConstants
```

Comment:
When changing the variables away from the enum, this could be changed to `from . import constants` and then the constants are used like `constants.NOISE_STREAM`.